lib/curl_sha512_256.c

```diff
@@ -292,7 +292,7 @@ Curl_sha512_256_finish(unsigned char *digest,
 #endif
 
 #if !defined(MHDX_INLINE)
-#  if defined(inline)
+#  if defined(inline) || defined(HAVE_INLINE_CHECKED)
```

Comment:
Then you can remove this ifdef completely

Reply:
Done